scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -155,7 +155,27 @@
   [dru]
   (* dru metrics-dru-scale))
 
-(defn compute-pending-job-dru
+(defn compute-pending-gpu-job-dru
```

Comment:
Add docstring for this function. It is probably worth putting a mention on why it exist separate from the function below.